<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_clientClass</class>
 <widget class="QMainWindow" name="qt_clientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Client Demo for Comm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_id">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineedit_id"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_pw">
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineedit_pw"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="push_login">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_token">
      <property name="text">
       <string>Token</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineedit_token">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_plate">
      <property name="text">
       <string>License Plate</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineedit_plate"/>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="push_query">
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPlainTextEdit" name="textedit_response"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ip">
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineedit_ip"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_client.qrc"/>
 </resources>
 <connections/>
</ui>
